images/build/Dockerfiles/gitHubRunners.BuildPackDepsBullseye.Dockerfile

```diff
@@ -0,0 +1,2 @@
+# https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md
+FROM buildpack-deps:bullseye-scm
```

Comment:
Should we define the SHA, as we do for Stretch/Buster, to make sure we're not broken if Debian pushes a new update to the base image?

Reply:
Thanks, that's good point, will update here soon once tested the sha locally

Reply:
Thanks!